types/node-fetch/index.d.ts

```diff
@@ -184,13 +184,14 @@ export interface ResponseInit {
     statusText?: string | undefined;
     timeout?: number | undefined;
     url?: string | undefined;
+    counter?: number | undefined;
 }
 
 interface URLLike {
     href: string;
 }
 
-export type HeadersInit = Headers | string[][] | { [key: string]: string };
+export type HeadersInit = Headers | string[][] | { [key: string]: string } | { [key: string]: string[] };
```

Comment:
This can be simplified by merging the two objects:
```suggestion
export type HeadersInit = Headers | string[][] | { [key: string]: string | string[] };
```

Reply:
Updated, thank you.